### ISSUE TITLE
2.5.5 & 2.5.8: How useful is target size when click area is not visible?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#3075]
The [intent of SCs for target size](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html) is:

> ... to help users who may have trouble activating a small target because of hand tremors, limited dexterity or other reasons.

However, the SCs only define a size for the target, but not visibility. In [my example](https://codepen.io/jaws-test/pen/bGxgyxY), the targets are large enough (50x50 px) to satisfy both SCs, but the visible targets are 
* too small (approx. 8x12 px)
* too close together

to meet the intent of the criterion.

Adjusting the SC 2.5.5 is too late. This is probably also true for 2.5.8, but I suggest to add at least in the Understanding a note:

> The click area of the targets should be visually visible (e.g. via their content, a border or a background color). If this is not possible, the visual content of the target should at least be in the middle of the click area.

**Comment by alastc** [comment]
There's an argument that, as a user, you'd probably aim for the right-hand side of the right-hand arrow, so ensure you don't hit the left one. Having the expanded hit area makes that more likely to work.

There are certainly cases where you can usefully expand the hit area beyond the visible part of the controls, that isn't something we would warn against in general.

**Comment by GreggVan** [comment]
What is the right hand side of the right hand arrow.

I think people usually point with the tip of the pointer   — or the tip of the finger when the hand cursor is used.   Is that not correct? 
(Note - my comment is ONLY from personal experience — not any research.  Hence it is a question to find out what research has found ) 


gregg

**Comment by patrickhlauke** [comment]
> What is the right hand side of the right hand arrow.

the example in the OP has two arrow icons. that's the arrows we're talking about, not the mouse pointer

**Comment by JAWS-test** [comment]
> There's an argument that, as a user, you'd probably aim for the right-hand side of the right-hand arrow, so ensure you don't hit the left one. Having the expanded hit area makes that more likely to work.

I think that most would try to click one of the two icons, because they assume that only these are clickable. Since the two icons are small and close together, there is a risk of accidentally hitting the other one.

**Comment by alastc** [comment]
Perhaps, but I think restricting hit areas to visible boundaries would be a net-negative.

**Comment by rachaelbradley** [comment]
I've added a short paragraph to https://github.com/w3c/wcag/pull/3147

**Comment by mbgower** [comment]
To answer the title of this issue:
>How useful is target size when click area is not visible?

It's not useful to anyone who uses vision to operate a web page, **including everyone who doesn't have a disability**.

It's important to state a reality:
- there is absolutely no requirement that an indicator for a target be _in the same location_ as the target.

So why aren't examples of missing or misplaced target indicators rampant on all web pages? Simply, a design that is difficult to use for most users is likely to be abandoned. I hope we don't need to spend too much time debating whether a invisible button -- or a visual button that was positioned at the opposite corner of the screen from its actual click area-- would not be easy to use.

Likewise, a visual indicator of a target that is not tightly tied in with the target is not going to prove very effective for any sighted user. 

For those reasons, I think it is unnecessary for us to specify such things. As well, It also doesn't seem that making such a specification would disproportionately benefit people with disabilities, for whom this specification is intended.
